src/components/actionBar/index.tsx

```diff
@@ -93,6 +99,25 @@ function ActionBar({ children, text, onClickBack, button }: Props) {
     );
   }
 
+  if (
+    !signerReady &&
+    exception &&
+    !location.pathname.includes(routes.gift.path)
+  ) {
+    const activeAddress =
+      defaultAccount.account?.cyber.name ||
```

Comment:
further I suggest to use defaultAccount.account?.cyber to avoid such snots

Reply:
We can remove cyber from the account object and use defaultAccount.account?

<img width="491" alt="image" src="https://github.com/cybercongress/cyb-ts/assets/36644677/a7e504e0-4ff2-461d-8a8c-ff53d468e1bb">